src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,86 @@
+package namor.template.calculator;
+
+import java.util.LinkedList;
+
+public class ClassicCalculator {
```

Comment:
Everything looks fine but you still was a bit copying previous solution that's why I'll give you additional little task. Look below